app/controllers/v1/projects_controller.rb

```diff
@@ -76,6 +76,30 @@ def destroy
       end
     end
 
+    swagger_api :list_tasks do
+      summary "List all of a project's tasks"
+      notes "This lists all of the selected project's the active tasks"
+      param :id
+    end
+    def list_tasks
+      project = Project.find_by(id: params[:id])
+      tasks = project.tasks
```

Comment:
I definitely can see the argument going both ways. I originally thought the former when I decided not to give tasks paging. I figured that most people probably wouldn't put enough tasks on a project to make them paging worthy. But I can definitely see now that it's an edge case of sorts. In the case a user wanted to make 20 or 100 tasks, it would be a better user experience to be able to page through them instead of scrolling down and down and down.